sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -150,7 +157,14 @@ internal open class KlaviyoApiRequest(
     /**
      * HTTP request headers
      */
-    override var headers: Map<String, String> = emptyMap()
+    override var headers: MutableMap<String, String> = mutableMapOf(
```

Comment:
should we enforce one or the other, either the field is mutable or the value itself it mutable? otherwise it might open itself to multiple forms of mutation which could get confusing, one place sets the header field vs another place that changes the header field?

Reply:
great point